### ISSUE TITLE
src/lib/arduino_lorawan_cEventLog.cpp - processSingleEvent

### ISSUE THREAD

**Issue by surfermarty** [mcci-catena/arduino-lorawan#191]
processSingleEvent is marked as returning bool, returns false if the queue is empty, but doesn't return anything at the end of the code path. Assume it should return true on/at line 106

**Comment by terrillmoore** [comment]
Yes, thanks. (At present, the function is `private`, and return result is never used by the only place it's called; so this is a latent but, rather than an actual fault. Still, this should be fixed.)
